src/server_manager/ui_components/outline-server-settings.html

```diff
@@ -202,6 +202,7 @@ <h3>[[localize('settings-metrics-header')]]</h3>
         serverHostname: {type: String, value: null},
         serverManagementApiUrl: {type: String, value: null},
         serverPortForNewAccessKeys: {type: Number, value: null},
+        isAccessKeyPortEditable: Boolean,
```

Comment:
Initalize explicitly: `{type: Bolean, value: false}`. Same for the one in server.

Reply:
Done